### DISCUSSION TITLE
Does delete local asset work?

### DISCUSSION THREAD

**Discussion by saulwold** [simulot/immich-go#158] [Q&A]
Hi There,

I was looking through the code saw that the Delete option of upload is commented out.  I started with trying to uncomment it, but still did not seem to actually delete the local file.

I am working through the code and trying to debug it, did not want to waste too much time if there is a know problem and someone is working on it already.

I can open an issue and work on it if you want some extra fingers in the code.

Sau!

**Comment by simulot** [comment]
You're right. Deleting local assets was in my intentions at the beginning. 

But, what if the upload fails silently and the original file is delete?  I felt not confortable with that idea. So have I edited the readme, and inactivated the code.

**Comment by hirenshah** [comment]
I can't seem to get immich's CLI working after the recent updates so was hoping to use immich go to replicate my previous setup.  

I drop files into an "import" folder and kick off a script which imports everything into immich and then deletes the source files.   Basically this, which is what the immich CLI offers:

`immich upload --recursive --delete /path/to/import`
